bindings/matrix-sdk-ffi/src/error.rs

```diff
@@ -288,6 +288,8 @@ pub enum RoomError {
     TimelineUnavailable,
     #[error("Invalid thumbnail data")]
     InvalidThumbnailData,
+    #[error("Invalid reply parameters")]
+    InvalidReplyParameters,
```

Comment:
Seems it's only used in one case when the event id is incorrect; can we specialize the error message to reflect that, please?